### PR TITLE
Cnft1 3308 UI add link to nbs 6 search

### PR DESCRIPTION
**Pull request by benlam-ignw**
## Description
A separate story will be created to add more detailed tests in `apps/modernization-ui/src/design-system/extendedTooltip/ExtendedTooltip.spec.tsx`. 

Patient search header vs the existing search navigation component conditional rendering is based off of `features?.search?.events?.enabled`

**UPDATED:**
Screenshot 2024-10-24 at 11 52 08 AM

**OLD:**

Screenshot 2024-10-23 at 10 12 19 AM

## Tickets

https://cdc-nbs.atlassian.net/browse/CNFT1-3308

## Checklist before requesting a review
- [x] PR focuses on a single story
- [x] Code has been fully tested to meet acceptance criteria
- [x] PR is reasonably small and reviewable (Generally less than 10 files and 500 changed lines)
- [x] All new functions/classes/components reasonably small
- [x] Functions/classes/components focused on one responsibility
- [x] Code easy to understand and modify (clarity over concise/clever)
- [x] PRs containing TypeScript follow the [Do's and Don'ts](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html)
- [x] PR does not contain hardcoded values (Uses constants)
- [ ] All code is covered by unit or feature tests

### CODE REVIEW

File: apps/modernization-ui/src/design-system/extendedTooltip/ExtendedTooltip.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import styles from './extended-tooltip.module.scss';
+
+type Props = {
+    labelTitle: string;
+    labelText: string;
+    position?: 'bottom' | 'top' | 'left' | 'right' | undefined;
+    children: ReactNode;
+};
+
+const ExtendedTooltip = ({ labelTitle, labelText, position, children }: Props) => {
```
Comment (adamloup-enquizit) [inline apps/modernization-ui/src/design-system/extendedTooltip/ExtendedTooltip.tsx:12]: suggestion:  Could you rename this to `RichTooltip`?  That is what the design team is referring to this component as.  

question: Isn't`labelText` the same as `children` here?

Comment (benlam-ignw) [reply apps/modernization-ui/src/design-system/extendedTooltip/ExtendedTooltip.tsx:12]: 1) Sure thing! 
2) Label title and label text are used in the label that gets shown/hidden in the USWDS Tooltip component, while the children like the info outlined icon here are always shown

Comment (adamloup-enquizit) [reply apps/modernization-ui/src/design-system/extendedTooltip/ExtendedTooltip.tsx:12]: Ah, that makes sense

**Comment by mpeels** [comment]
Issue: the spacing seems a bit off for the text / icon from Figma
Figma: 
image

**Review by adamloup-enquizit — COMMENTED** [review]
The placement of the rich tool tip looks a little off, the designs have place under the icon with the left side of the tooltip aligned with it vertically.  I don't think that little arrow should be their either.

**Comment by benlam-ignw** [comment]
Closing for now while I rewrite the RichTooltip component

**Comment by benlam-ignw** [comment]
I've created custom tooltip logic in the `Rich Tooltip` component, please check it out and let me know if you have any feedback :)

File: apps/modernization-ui/src/design-system/richTooltip/rich-tooltip.module.scss [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  margin: 0;
+  padding: 16px;
+  font-size: 14px;
+  line-height: 20px;
+  background-color: white;
+  color: black;
+  outline-color: #E6E7E8;
+  outline-width: 1px;
+  outline-style: solid;
+  filter: drop-shadow(0px 4px 2px #9f9f9f);
```
Comment (mpeels) [inline apps/modernization-ui/src/design-system/richTooltip/rich-tooltip.module.scss:12]: Question: Are these colors defined in the `styles/_colors.scss`? If not, might want to confirm with designer about using one that is already defined.

Comment (benlam-ignw) [reply apps/modernization-ui/src/design-system/richTooltip/rich-tooltip.module.scss]: forgot to fix this, thanks!

**Review by mpeels — APPROVED** [review]

**Review by hclarkEnq — APPROVED** [review]

**Pull request merged**